ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt

```diff
@@ -30,8 +78,62 @@ hideall "--sync--"
 #6000.0 "--Reset--" sync / 00:0839:Pride of the Lioness is no longer sealed/ window 0,2000 jump 0
 
 ### Dahu
+# -p 576D:8008.4
+# -ii 576C 5B43
 8000.0 "--sync--" sync / 00:0839:The Hall of Supplication will be sealed off/ window 10000,0
 8000.0 "--Reset--" sync / 00:0839:The Hall of Supplication is no longer sealed/ window 0,2000 jump 0
+8008.4 "Reverberating Roar" sync /:Dahu:576D:/ window 10,2.5
+8013.5 "Falling Rock x6" duration 10 #sync /:Dahu:576E:/
+8014.5 "Reverberating Roar" sync /:Dahu:576D:/
+8022.7 "Hot Charge 1" sync /:Dahu:5773:/
+8027.5 "Hot Charge 2" sync /:Dahu:5773:/
+8034.3 "Firebreathe" sync /:Dahu:5774:/
+8046.5 "--sync--" sync /:Marchosias:576A:/
+8051.8 "Head Down" sync /:Marchosias:5768:/
+8057.5 "Head Down" sync /:Marchosias:5768:/
+8061.4 "Spit Flame" sync /:Dahu:5776:/
+8061.6 "Spit Flame 1" #sync /:Dahu:5777:/
+8062.7 "Spit Flame 2" #sync /:Dahu:5777:/
+8063.8 "Spit Flame 3" #sync /:Dahu:5777:/
+8065.0 "Spit Flame 4" #sync /:Dahu:5777:/
+8069.1 "Head Down" sync /:Marchosias:5768:/
+8071.1 "Right-Sided Shockwave/Left-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+8073.7 "Left-Sided Shockwave/Right-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+8074.8 "Head Down" sync /:Marchosias:5768:/
+8081.4 "Feral Howl" sync /:Dahu:5767:/
+8083.5 "--knockback--" sync /:Dahu:5B35:/ # TODO: confirm
+8084.9 "Hunter's Claw" sync /:Marchosias:5769:/
+8091.5 "Firebreathe x5" sync /:Dahu:576B:/ duration 8.7
+8112.5 "--sync--" sync /:Crowned Marchosias:576A:/
+8117.5 "--sync--" sync /:Crowned Marchosias:576A:/
+8122.5 "--sync--" sync /:Crowned Marchosias:576A:/
+8142.5 "--sync--" sync /:Marchosias:576A:/
```

Comment:
I think 576A is the spawning of the outer lions. I called it "--lions--", but that is easy to mix-up with the adds of 576A.

Reply:
Is it important to call those out, or is it better to leave as sync?

Reply:
I think the lions is not important to call out and can be left as a sync. The lions will be moving around and so awareness for their position is not needed until Spirit Flame mechanic or knockback mechanics to void Hunter's Claws.

Knowing where the lions are for Heads Down is not critical in this fight, there is plenty of time given for adjusting to their position and aoe markers.